modules/blocks.js

```diff
@@ -252,14 +329,17 @@ __private.getIdSequence = function (height, cb) {
 			}
 		}
 
+		// Add last block at the beginning if set not cointains it already
```

Comment:
// Add last block at the beginning if set not cointains it already

Add last block at the beginning if the set does not contain it already